include/xframe/xselecting.hpp

```diff
@@ -39,6 +41,19 @@ namespace xf
             }
         };
 
+        template <class T, class B>
+        struct static_missing_impl<xt::xmasked_value<T, B>>
+        {
+            using return_type = xt::xmasked_value<T, B>;
+            static inline return_type get()
+            {
+                static B has_val = false;
+                static T val = T(0, has_val);
```

Comment:
I am not 100% sure about this one. Here I am expecting `T` to be an `xoptional`, maybe I should not.

Reply:
Yeah I'm not really sure about a `static_missing` returning an `xmasked_value` of `xoptional`, it does not feel right. Which made me think of this: #194

Reply:
I think it's valid, but there should not be any assumption on T; so this lines should be replaced with:
```cpp
static T val = static_missing_impl<T>::get();
```